passhole/passhole.py

```diff
@@ -596,24 +598,38 @@ def call_xdotool(args):
 
     # type out password
     k = Controller()
-    if args.tabbed:
-        if selected_entry.username:
+    if args.totp:
```

Comment:
Maybe put this in an elif instead of nesting?

Reply:
Hi, I have pushed the code with the suggested changes. I have made sure that the password is not typed when --totp is sent as an argument.